GDJS/Runtime/pixi-renderers/layer-pixi-renderer.js

```diff
@@ -65,7 +65,7 @@ gdjs.LayerPixiRenderer.prototype.updateVisibility = function (visible) {
 gdjs.LayerPixiRenderer.prototype.updateTime = function () {
   if (this._renderTexture) {
     this.updateRenderTexture();
-    this.syncWithBaseLayer();
+    this._layer.syncWithBaseLayer();
   }
```

Comment:
I realized `syncWithBaseLayer` shouldn't be called inside this if block, maybe there would be some other condition like "automatically follow base layer" which makes more sense.

Reply:
Yes that's why I think it's a good thing to put this in gdjs.Layer - the option to  "automatically follow base layer" (which can be a checkbox somewhere in the UI, as well as a condition and action to test/enable/disable it) is not tied to Pixi and should be working entirely from `gdjs.Layer`. In this case, `update` seems like the best candidate where to do this :)